deployment/v3/mosip/resident/install.sh

```diff
@@ -7,7 +7,7 @@ if [ $# -ge 1 ] ; then
 fi
 
 NS=resident
-CHART_VERSION=12.0.1-B2
+CHART_VERSION=12.0.1-B3
```

Comment:
mimoto will be deployed with 12.0.1-B3
and resident, resident UI will deployed with 12.0.1-B2

CHART_VERSION=12.0.1-B2
MIMOTO_CHART_VERSION=12.0.1-B3